pandas/_libs/groupby.pyx

```diff
@@ -1259,7 +1274,10 @@ cdef group_min_max(groupby_t[:, ::1] out,
                         runtime_error = True
                         break
                     else:
-                        out[i, j] = nan_val
+                        if uses_mask:
+                            result_mask[i, j] = True
+                        else:
+                            out[i, j] = nan_val
```

Comment:
How is `out` typically initialized? (`np.empty`?) I am wondering it it would be good practice to sill set `out` anyway (also if `uses_mask=True` to not have "unitialized" values)

Reply:
hmm i could go either way on this